### ISSUE TITLE
Version 6.5.4's MSDF font rendering is broken

### ISSUE THREAD

**Issue by insraq** [pixijs/pixijs#8686]
![image](https://user-images.githubusercontent.com/608221/193354388-a08218de-40c5-4c24-9be2-3ded46c21693.png)

I believe this PR breaks MSDF font rendering: https://github.com/pixijs/pixijs/pull/8652/

You can see in 6.5.3, the MSDF font is rendered very crispy and clear. After upgrading to 6.5.4, the font rendering is very blurry.

**Comment by bigtimebuddy** [comment]
@ivanpopelyshev want to weigh-in on this? Also @insraq could you please provide some code/reproduction here? Is this because you're using resolution != 1?

**Comment by insraq** [comment]
Actually, simple use the examples in the PR: 
https://codesandbox.io/s/pixi-v6-bitmap-text-gray-square-rsp91x
or https://codesandbox.io/s/pixi-v6-bitmap-text-gray-square-fix-bf5k4l

Bump pixi version to 6.5.4 and you can see the font is blurry 

![image](https://user-images.githubusercontent.com/608221/193355778-634d0487-2cb4-493a-af42-9327f7587be7.png)

Although this might be broken by other PRs between 6.5.3 and 6.5.4 - but PR #8652 is the most related I can find from the changelog

**Comment by bigtimebuddy** [comment]
I think you're right about the PR. We'll wait for Ivan to respond.

**Comment by ivanpopelyshev** [comment]
Thank you! gonna check that

**Comment by saitonakamura** [comment]
Yeah, it looks like those grey rectangles, but not over the whole symbol, rather on the edges
<img width="340" alt="image" src="https://user-images.githubusercontent.com/1552189/193773175-23eb046d-c5e3-478b-b513-97819b35a690.png">

**Comment by ivanpopelyshev** [comment]
I know how to fix that: instead of using `min(width, 1.0)` in shader, we have to check texture for 0 and 1 values.

**Comment by bigtimebuddy** [comment]
@ivanpopelyshev PR?

**Comment by ivanpopelyshev** [comment]
After today's https://www.khronos.org/events/webgl-webgpu-meetup-october-4-2022

**Comment by ivanpopelyshev** [comment]
@insraq  can you try https://codesandbox.io/s/pixi-v6-bitmap-text-gray-square-fix-2-9y10bi?file=/src/CoolBitmapText.js in your project?

**Comment by bigtimebuddy** [comment]
Merged fix #8699

**Comment by insraq** [comment]
Sorry I was busy with other stuff. I can confirm https://github.com/pixijs/pixijs/pull/8699 does fix the issue for me. Thanks!

**Comment by ivanpopelyshev** [comment]
Its magic! I dont know why was it wrong!
